### PR TITLE
end2end tutorial segmenter and summerizer

### PR DESCRIPTION
**Pull request by SatyamRajawat**

### CODE REVIEW

File: jaseci_kit/jaseci_kit/modules/summarization/README.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    jsctl -m
+    ```
+2.  Load cl_summer module in jac by command
+    ```
+    actions load module jaseci_kit.cl_summer
+    ```
+
+
+## **2. Summarizer**
+For this tutorial, we are going to leverage the **Summarizer** (`cl_summer`) which would be generate the summary from text.
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/summarization/README.md]: which would help us to generate summary of the provided text.

File: jaseci_kit/jaseci_kit/modules/summarization/README.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+            
+            report cl_summer.summarize(
+                text = data["text"],
+                url = data["url"],
+                sent_count = data["sent_count"],
+                summarizer_type = data["summarizer_type"]
+                );      
+        }
+        ```
+        `summarize`: to get the extractive summary in provided sentences count.
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/summarization/README.md]: to get the extractive summary of the text in the given number of sentence counts .

File: jaseci_kit/jaseci_kit/modules/summarization/README.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+        **Parameter details**
+        * Input
+            * `model_name(String)`: name of the transformer model to load, options are:
+                * `wiki` : trained on wikipedia data
+                * `legal`: trained on legal documents
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/summarization/README.md]: Model name content is wrong should be removed

File: jaseci_kit/jaseci_kit/modules/t5_sum/README.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
+# **Summarization(`t5_sum`)**
+
+Module `t5_sum` uses the hugging face T5 transformer model to extract summary from text.
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/t5_sum/README.md]: transformer model to provide abstractive summary from text.

File: jaseci_kit/jaseci_kit/modules/text_segmenter/README.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+                        "He watched her as she toiled slowly and with great care. Six times she tried to throw her frail thread from one beam to another, and six times it fell short. '",
+                        "Poor thing,' said Bruce: 'you, too, know what it is to fail.",
+                        "But the spider did not lose hope with the sixth failure.",
+                        "With still more care, she made ready to try for the seventh time.",
+                        "Bruce almost forgot his own troubles as he watched her swing herself out upon the slender line.",
+                        "Would she fail again?",
+                        "No! The thread was carried safely to the beam and fastened there."
+                        ]
+                    ]
+            ```
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/text_segmenter/README.md:213]: change the output

**Review by AshishMahendra — CHANGES_REQUESTED** [review]

File: jaseci_kit/jaseci_kit/modules/summarization/README.md (hunk 1 above)
Comment (SatyamRajawat) [reply jaseci_kit/jaseci_kit/modules/summarization/README.md]: done

File: jaseci_kit/jaseci_kit/modules/summarization/README.md (hunk 2 above)
Comment (SatyamRajawat) [reply jaseci_kit/jaseci_kit/modules/summarization/README.md]: done

File: jaseci_kit/jaseci_kit/modules/summarization/README.md (hunk 3 above)
Comment (SatyamRajawat) [reply jaseci_kit/jaseci_kit/modules/summarization/README.md]: done

File: jaseci_kit/jaseci_kit/modules/t5_sum/README.md (hunk 4 above)
Comment (SatyamRajawat) [reply jaseci_kit/jaseci_kit/modules/t5_sum/README.md]: done

File: jaseci_kit/jaseci_kit/modules/text_segmenter/README.md (hunk 5 above)
Comment (SatyamRajawat) [reply jaseci_kit/jaseci_kit/modules/text_segmenter/README.md:213]: done

**Pull request merged**